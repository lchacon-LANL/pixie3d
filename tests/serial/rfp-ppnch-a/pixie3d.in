 &datin
    br_pert_phase_old=   0.0000000000000000
    br_pert_bc_old=   0.0000000000000000
    E0=   0.00000000     ,   0.00000000     ,   4.00000019E-03
    ilevel   = 0
    test     = t

    nxd      = 128
    nyd      = 32 
    nzd      = 1

    xmax     = 1.
    ymax     = 0.
    zmax     = 1.

    numerical_grid = f

    bcs      = 'spt','def','per','per','per','per'

    eta      = 1d-3
    nu       = 1d-3
    dd       = 2d-3
    chi      = 0d-2
    
    equil    = 'ppnch'

    coords   = 'hel'

    dlambda  = 0.5
    gparams  = 1d0,-2.0d0,4d0,0d0,4d0,1d0

    gamma    = 1d0

    dt       = 5d-3
    tmax     = 5d-2
    dstep    = 5d-2

    pvx      = 1d-4
    odd      = f,f,f
    random   = f
    nh2      = 1

    iguess   = 0

    cnfactor = 0.5

    restart  = f
    timecorr = f

    source   = t

    precon   = 'id'

    maxksp   = 20
    maxitgm  = 20
    tolgm    = 8d-1
    rtol     = 1d-4
    atol     = 0d0
    method   = 1
    mf_eps   = 1d-6
    vol_wgt  = t

    nc_eom_f = t
    nc_eom_v = f
    solve_rho= f

 /

 &graphdef
    sel_diag  = 2,12,15,17,18,19,20,22,23
    sel_graph = 1,2,3,5,6,7,9,10,11
    dplot = 1d-3
    prof_conf = 1,'r',1,10,1
    cont_conf = 3,'x','y',1,1,1
    hdf_plot = f
    clean    = f
 /

