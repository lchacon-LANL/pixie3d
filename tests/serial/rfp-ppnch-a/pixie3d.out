 Calculating equilibrium ppnch...

 ***************************
 1D Ohmic equilibrium solver
 ***************************

 1D equilibrium converged in   8 iterations with error =  4.08E-05

 Force-balance error = 2.7818097E-03
 Ohms law      error = 7.6747232E-03

 Pinch equilibrium features:
   Theta  = 1.6518018
   Bz(r=0)= 1.0000000
   I0     = 2.0634714
   E0/eta = 4.0000000
   T_flux = 0.6246123
 A-solve GMRES its= 13

  Grid mesh:................   32x   32x    1
  Number of grid levels:....    4,    4,    1
  Preconditioning method:... Identity
  Time correction...........   F
  External source...........   F
  Numerical grid............   T
  Scalar advection scheme...  SMART   
  Velocity advection scheme.  SMART   
  TS scheme................. Theta    

   itime    time     dt     #CFL   vx_max   vy_max   vz_max    GMRES Newton CN factor
      1    0.010  1.0E-02 7.8E+00  1.1E-02  1.4E-06  3.3E-08     5     4     0.500
      2    0.020  1.0E-02 7.8E+00  1.1E-02  5.0E-06  1.3E-07     6     5     0.500
      3    0.030  1.0E-02 7.8E+00  1.1E-02  9.9E-06  3.0E-07     6     4     0.500
      4    0.040  1.0E-02 7.8E+00  1.1E-02  1.5E-05  5.3E-07     7     5     0.500
      5    0.050  1.0E-02 7.8E+00  1.1E-02  2.1E-05  8.3E-07     9     5     0.500
      6    0.060  1.0E-02 7.8E+00  1.1E-02  2.8E-05  1.2E-06     8     5     0.500
      7    0.070  1.0E-02 7.8E+00  1.1E-02  3.6E-05  1.6E-06     8     5     0.500
      8    0.080  1.0E-02 7.8E+00  1.1E-02  4.5E-05  2.1E-06    10     5     0.500
      9    0.090  1.0E-02 7.8E+00  1.1E-02  5.6E-05  2.7E-06     9     5     0.500
     10    0.100  1.0E-02 7.8E+00  1.1E-02  7.0E-05  3.3E-06    10     5     0.500

Final statistics

 itime  Newt/ndt  GMRES/ndt  GMRES/Newt
   10       4.8        7.8        1.6
